checks/raw/fuzzing.go

```diff
@@ -15,14 +15,44 @@
 package raw
 
 import (
+	"bytes"
 	"fmt"
+	"regexp"
+	"strings"
 
 	"github.com/ossf/scorecard/v4/checker"
 	"github.com/ossf/scorecard/v4/checks/fileparser"
 	"github.com/ossf/scorecard/v4/clients"
 	sce "github.com/ossf/scorecard/v4/errors"
 )
 
+const (
+	FuzzNameOSSFuzz         = "OSS-Fuzz"
+	FuzzNameClusterFuzzLite = "ClusterFuzzLite"
+	FuzzNameUserDefinedFunc = "user-defined fuzz functions"
```

Comment:
Let's make the naming for user-defined functions something to the effect of:
```
FuzzNameGoBuiltIn = "GoBuiltInFuzzer"
```

Reply:
The issue now is, we treat each fuzz func as a file and put files of different programming langs into one `[]File` array. Since both `[]File`, `URL`, 'Desc' are params of a `Tool`, it would be confusing when a repo has multiple prominent langs (although now I just implemented the check for golang).

Should I still use the doc, url, and fuzzer name of golang for the `Tool` obj? Or should I separate files of diff langs into multiple `Tool` objs?

Reply:
we can have one fuzzer per language for these. They are different fuzzers